admin/tests/unit/services/version.js

```diff
@@ -13,53 +23,59 @@ describe('version', () => {
     });
 
     it('A beta returns the beta incremented by one', () => {
-      chai.expect(service.minimumNextRelease('1.2.3-beta.0')).to.deep.equal({
-        major: 1,
-        minor: 2,
-        patch: 3,
-        beta: 1
-      });
-      chai.expect(service.minimumNextRelease('1.2.3-beta.1')).to.deep.equal({
-        major: 1,
-        minor: 2,
-        patch: 3,
-        beta: 2
-      });
-      chai.expect(service.minimumNextRelease('1.2.3-beta.9')).to.deep.equal({
-        major: 1,
-        minor: 2,
-        patch: 3,
-        beta: 10
-      });
-      chai.expect(service.minimumNextRelease('1.2.3-beta.10')).to.deep.equal({
-        major: 1,
-        minor: 2,
-        patch: 3,
-        beta: 11
-      });
+      chai.expect(service.minimumNextRelease('1.2.3-beta.0')).to.deep.equal(
+        v(1,2,3,1));
```

Comment:
Any reason for these to be on the next line? I find it a little bit harder to read but just personal preference.

Reply:
No major reason, apart from code length. I'll change it here though, you're right it's easier to read and not that egregious.